iroha/src/sumeragi/mod.rs

```diff
@@ -264,14 +290,20 @@ impl<Q: QueueTrait, G: GenesisNetworkTrait, W: WorldTrait> ContextHandler<Init>
             height,
         }: Init,
     ) {
+        iroha_logger::info!("Starting Sumeragi");
+        self.connect_peers().await;
+        time::sleep(Duration::from_millis(300)).await;
```

Comment:
Kinda dont like sleeps at random places. We can do the same without them...

Reply:
Removed.